### PR TITLE
Update default config

### LINKED ISSUE #80: How to ignore system generated annotations?
Some annotations and attributes are added by system. Deploy from `kapp` seems deleting them.

```
--- update persistentvolumeclaim/juicefs-pvc (v1) namespace: chaos-sut
  ...
  2,  2   metadata:
  3     -   annotations:
  4     -     pv.kubernetes.io/bind-completed: "yes"
  5     -     pv.kubernetes.io/bound-by-controller: "yes"
  6     -     volume.beta.kubernetes.io/storage-provisioner: csi.juicefs.com
  7,  3     creationTimestamp: "2020-03-08T03:47:50Z"
  8,  4     finalizers:
  ...
 24, 20     storageClassName: juicefs-sc
 25     -   volumeMode: Filesystem
 26, 21     volumeName: pvc-7eb42232-0567-4115-a5cc-4c48af07bf66
 27, 22   status:
```

How can I keep them unchanged?

### PR DESCRIPTION
**Pull request by yujunz**
Close #80

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@yujunz Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=k14s/kapp&pullRequestId=101)!

[Click here](https://cla.pivotal.io/sync/pivotal?repositoryId=k14s/kapp&pullRequestId=101) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@yujunz Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/pivotal?repositoryId=k14s/kapp&pullRequestId=101)!

File: pkg/kapp/config/default.go [hunk 1]
```diff
@@ -72,21 +72,39 @@ rebaseRules:
...
       kind: PersistentVolumeClaim
 
+- path: [metadata, annotations, pv.kubernetes.io/bound-by-controller]
+  type: copy
+  sources: [existing]
+  resourceMatchers: *pvcs
+
+- path: [metadata, annotations, volume.beta.kubernetes.io/storage-provisioner]
+  type: copy
+  sources: [existing]
```
Comment (cppforlife) [inline pkg/kapp/config/default.go:91]: i think it's still worth keeping `[new, existing]` for this one and above. wdyt?

Comment (yujunz) [reply pkg/kapp/config/default.go]: This is actually copied from https://github.com/k14s/kapp/blob/develop/docs/rebase-pvc.md Shall we update that as well?

Comment (cppforlife) [reply pkg/kapp/config/default.go:91]: probably.

Comment (yujunz) [reply pkg/kapp/config/default.go]: Done for both

**Pull request merged**